css/sketchpad.css

```diff
@@ -1,23 +1,42 @@
+html * {
+    box-sizing: border-box;
+}
+
 /*.container height in percentage wil not work unless
 its parent element has a specified height.
 */
 body {
-    /*height: 80vh;*/
     background-color: #DCDDD6;
     position: relative;
-    margin: 40 auto;
+    margin: 20px;
     font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
 }
 
+/* apply default clearfix styles on body -- because body contains
+floated elements */
+body:after {
+    content:"";
+    display:table;
+    clear:both;
+}
+
 .container {
     height: 980px;
-    width: 980px;
     float: right;
     position: relative;
-    margin: 0px 20px auto;
+    margin: 0;
+}
+
+/* side bar created for button elements to sit*/
+aside {
+    background-color: #edf699;
+    float: left;
+    height: 980px;
```

Comment:
Hard-coding heights in multi-column layouts is generally a bad idea. Can you find another way for the next minor version release of your project? ;)